File: java/src/main/java/api/cornell/data/classes/Subject.java
package api.cornell.data.classes;

import org.jetbrains.annotations.NotNull;

/**
 * {@code Subject} is a collection of supported subjects value.
 */
public enum Subject {
    
    /**
     * Asian American Studies.
     */
    AAS("Asian American Studies"),
    /**
     * American Indian Studies.
     */
    AIS("American Indian Studies"),
    /**
     * Applied Economics & Management.
     */
    AEM("Applied Economics & Management"),
    /**
     * Applied & Engineering Physics.
     */
    AEP("Applied & Engineering Physics"),
    /**
     * Agriculture Sciences.
     */
    AGSCI("Agriculture Sciences"),
    /**
     * American Indian & Indigenous.
     */
    AIIS("American Indian & Indigenous"),
    /**
     * Air Force Science.
     */
    AIRS("Air Force Science"),
    /**
     * Akkadian.
     */
    AKKAD("Akkadian"),
    /**
     * Agriculture & Life Sciences.
     */
    ALS("Agriculture & Life Sciences"),
    /**
     * American Studies.
     */
    AMST("American Studies"),
    /**
     * Animal Science.
     */
    ANSC("Animal Science"),
    /**
     * Anthropology.
     */
    ANTHR("Anthropology"),
    /**
     * Arabic.
     */
    ARAB("Arabic"),
    /**
     * Architecture.
     */
    ARCH("Architecture"),
    /**
     * Archaeology.
     */
    ARKEO("Archaeology"),
    /**
     * Art.
     */
    ART("Art"),
    /**
     * Art History.
     */
    ARTH("Art History"),
    /**
     * Arts & Sciences.
     */
    AS("Arts & Sciences"),
    /**
     * Asian Studies.
     */
    ASIAN("Asian Studies"),
    /**
     * Africana Studies & Res Center.
     */
    ASRC("Africana Studies & Res Center"),
    /**
     * Astronomy.
     */
    ASTRO("Astronomy"),
    /**
     * Bosnian, Croatian, Serbian.
     */
    BCS("Bosnian, Croatian, Serbian"),
    /**
     * Biological & Environmental Eng.
     */
    BEE("Biological & Environmental Eng"),
    /**
     * Bengali.
     */
    BENGL("Bengali"),
    /**
     * Animal Physiology & Anatomy.
     */
    BIOAP("Animal Physiology & Anatomy"),
    /**
     * Ecology & Evolutionary Biology.
     */
    BIOEE("Ecology & Evolutionary Biology"),
    /**
     * Biology: General Courses.
     */
    BIOG("Biology: General Courses"),
    /**
     * Molecular Biology and Genetics.
     */
    BIOMG("Molecular Biology and Genetics"),
    /**
     * Microbiology.
     */
    BIOMI("Microbiology"),
    /**
     * Bio Medical Science.
     */
    BIOMS("Bio Medical Science"),
    /**
     * Plant Biology.
     */
    BIOPL("Plant Biology"),
    /**
     * Neurobiology & Behavior.
     */
    BIONB("Neurobiology & Behavior"),
    /**
     * Biomedical Engineering.
     */
    BME("Biomedical Engineering"),
    /**
     * Baker Program in Real Estate.
     */
    BPRE("Baker Program in Real Estate"),
    /**
     * Biology & Society.
     */
    BSOC("Biology & Society"),
    /**
     * Biometry & Statistics.
     */
    BTRY("Biometry & Statistics"),
    /**
     * Burmese.
     */
    BURM("Burmese"),
    /**
     * China & Asia Pacific Studies.
     */
    CAPS("China & Asia Pacific Studies"),
    /**
     * Civil & Environmental Engr.
     */
    CEE("Civil & Environmental Engr"),
    /**
     * Chemistry.
     */
    CHEM("Chemistry"),
    /**
     * Chemical Engineering.
     */
    CHEME("Chemical Engineering"),
    /**
     * Chinese.
     */
    CHIN("Chinese"),
    /**
     * Chinese Literature.
     */
    CHLIT("Chinese Literature"),
    /**
     * Classics.
     */
    CLASS("Classics"),
    /**
     * Cognitive Science.
     */
    COGST("Cognitive Science"),
    /**
     * College Scholar Program.
     */
    COLLS("College Scholar Program"),
    /**
     * Comparative Literature.
     */
    COML("Comparative Literature"),
    /**
     * Communication.
     */
    COMM("Communication"),
    /**
     * City & Regional Planning.
     */
    CRP("City & Regional Planning"),
    /**
     * Computer Science.
     */
    CS("Computer Science"),
    /**
     * Crop & Soil Sciences.
     */
    CSS("Crop & Soil Sciences"),
    /**
     * Czech.
     */
    CZECH("Czech"),
    /**
     * Design & Environmental Analy.
     */
    DEA("Design & Environmental Analy"),
    /**
     * Development Sociology.
     */
    DSOC("Development Sociology"),
    /**
     * Dutch.
     */
    DUTCH("Dutch"),
    /**
     * Earth & Atmospheric Sciences.
     */
    EAS("Earth & Atmospheric Sciences"),
    /**
     * Electrical & Computer Engineering.
     */
    ECE("Electrical & Computer Engineering"),
    /**
     * Economics.
     */
    ECON("Economics"),
    /**
     * Education.
     */
    EDUC("Education"),
    /**
     * English Language Support.
     */
    ELSO("English Language Support"),
    /**
     * English.
     */
    ENGL("English"),
    /**
     * Engineering Communications.
     */
    ENGRC("Engineering Communications"),
    /**
     * Engineering Distribution.
     */
    ENGRD("Engineering Distribution"),
    /**
     * Engineering General Interest.
     */
    ENGRG("Engineering General Interest"),
    /**
     * Engineering Introduction.
     */
    ENGRI("Engineering Introduction"),
    /**
     * English for International Stdt.
     */
    ENINT("English for International Stdt"),
    /**
     * Engineering Management.
     */
    ENMGT("Engineering Management"),
    /**
     * Entomology.
     */
    ENTOM("Entomology"),
    /**
     * Envr and Sust Sciences.
     */
    ESS("Envr and Sust Sciences"),
    /**
     * Food Science.
     */
    FDSC("Food Science"),
    /**
     * Feminist,Gender,Sexuality Stdy.
     */
    FGSS("Feminist,Gender,Sexuality Stdy"),
    /**
     * French.
     */
    FREN("French"),
    /**
     * Fiber Science & Apparel Design.
     */
    FSAD("Fiber Science & Apparel Design"),
    /**
     * German Studies.
     */
    GERST("German Studies"),
    /**
     * Government.
     */
    GOVT("Government"),
    /**
     * Graduate Research.
     */
    GRAD("Graduate Research"),
    /**
     * Greek.
     */
    GREEK("Greek"),
    /**
     * Hotel Administration.
     */
    HADM("Hotel Administration"),
    /**
     * Human Development.
     */
    HD("Human Development"),
    /**
     * Human Ecology Nondepartmental.
     */
    HE("Human Ecology Nondepartmental"),
    /**
     * Hebrew.
     */
    HEBRW("Hebrew"),
    /**
     * Hindi.
     */
    HINDI("Hindi"),
    /**
     * History.
     */
    HIST("History"),
    /**
     * Horticulture Sciences.
     */
    HORT("Horticulture Sciences"),
    /**
     * Hungarian.
     */
    HUNGR("Hungarian"),
    /**
     * Intl Agriculture & Rural Dev.
     */
    IARD("Intl Agriculture & Rural Dev"),
    /**
     * ILR Human Resource Management.
     */
    ILRHR("ILR Human Resource Management"),
    /**
     * ILR International & Comp Labor.
     */
    ILRIC("ILR International & Comp Labor"),
    /**
     * ILR Interdepartmental.
     */
    ILRID("ILR Interdepartmental"),
    /**
     * ILR Labor Economics.
     */
    ILRLE("ILR Labor Economics"),
    /**
     * ILR Labor Relations, Law, Hist.
     */
    ILRLR("ILR Labor Relations, Law, Hist"),
    /**
     * ILR Organizational Behavior.
     */
    ILROB("ILR Organizational Behavior"),
    /**
     * ILR Social Statistics.
     */
    ILRST("ILR Social Statistics"),
    /**
     * Independent Major.
     */
    IM("Independent Major"),
    /**
     * Indonesian.
     */
    INDO("Indonesian"),
    /**
     * Information Science.
     */
    INFO("Information Science"),
    /**
     * Italian.
     */
    ITAL("Italian"),
    /**
     * Japanese.
     */
    JAPAN("Japanese"),
    /**
     * Japanese Literature.
     */
    JPLIT("Japanese Literature"),
    /**
     * Jewish Studies.
     */
    JWST("Jewish Studies"),
    /**
     * Kannada.
     */
    KANAD("Kannada"),
    /**
     * Khmer.
     */
    KHMER("Khmer"),
    /**
     * Korean.
     */
    KOREA("Korean"),
    /**
     * Korean Literature.
     */
    KRLIT("Korean Literature"),
    /**
     * Landscape Architecture.
     */
    LA("Landscape Architecture"),
    /**
     * Latin American Studies.
     */
    LATA("Latin American Studies"),
    /**
     * Latin.
     */
    LATIN("Latin"),
    /**
     * Law.
     */
    LAW("Law"),
    /**
     * Leadership.
     */
    LEAD("Leadership"),
    /**
     * Lesbian, Gay, Bisexual, Trns Stdy.
     */
    LGBT("Lesbian, Gay, Bisexual, Trns Stdy"),
    /**
     * Linguistics.
     */
    LING("Linguistics"),
    /**
     * Latino Studies Program.
     */
    LSP("Latino Studies Program"),
    /**
     * Mechanical & Aerospace Eng.
     */
    MAE("Mechanical & Aerospace Eng"),
    /**
     * Mathematics.
     */
    MATH("Mathematics"),
    /**
     * Medieval Studies.
     */
    MEDVL("Medieval Studies"),
    /**
     * Military Science.
     */
    MILS("Military Science"),
    /**
     * Modern Greek.
     */
    MOGRK("Modern Greek"),
    /**
     * Materials Science & Engr.
     */
    MSE("Materials Science & Engr"),
    /**
     * Music.
     */
    MUSIC("Music"),
    /**
     * Grad Mgmt Acct.
     */
    NACCT("Grad Mgmt Acct"),
    /**
     * Naval Science.
     */
    NAVS("Naval Science"),
    /**
     * Grad Mgmt Business Admin.
     */
    NBA("Grad Mgmt Business Admin"),
    /**
     * Grad Mgmt Business Admin CT
     */
    NBAT("Grad Mgmt Business Admin CT"),
    /**
     * Executive Boardroom Electives.
     */
    NBAB("Executive Boardroom Electives"),
    /**
     * Business Admin Electives EMBA.
     */
    NBAE("Business Admin Electives EMBA"),
    /**
     * Grad Mgmt Business Admin NYT.
     */
    NBAY("Grad Mgmt Business Admin NYT"),
    /**
     * Grad Mgmt Common Core.
     */
    NCC("Grad Mgmt Common Core"),
    /**
     * Executive Boardroom Core.
     */
    NCCB("Executive Boardroom Core"),
    /**
     * Grad Mgmt Common Core NYT.
     */
    NCCY("Grad Mgmt Common Core NYT"),
    /**
     * Nepali.
     */
    NEPAL("Nepali"),
    /**
     * Near Eastern Studies.
     */
    NES("Near Eastern Studies"),
    /**
     * Grad Mgmt Individual Study.
     */
    NMI("Grad Mgmt Individual Study"),
    /**
     * Graduate Management Research.
     */
    NRE("Graduate Management Research"),
    /**
     * Nutritional Science.
     */
    NS("Nutritional Science"),
    /**
     * Nuclear Science & Engineering.
     */
    NSE("Nuclear Science & Engineering"),
    /**
     * Natural Resources.
     */
    NTRES("Natural Resources"),
    /**
     * Op Research & Information Engr.
     */
    ORIE("Op Research & Information Engr"),
    /**
     * Public Administration.
     */
    PADM("Public Administration"),
    /**
     * Pali.
     */
    PALI("Pali"),
    /**
     * Policy Analysis & Management.
     */
    PAM("Policy Analysis & Management"),
    /**
     * Physical Education & Athletics.
     */
    PE("Physical Education & Athletics"),
    /**
     * Persian.
     */
    PERSN("Persian"),
    /**
     * Philosophy.
     */
    PHIL("Philosophy"),
    /**
     * Physics.
     */
    PHYS("Physics"),
    /**
     * Plant Biology.
     */
    PLBIO("Plant Biology"),
    /**
     * Plant Breeding.
     */
    PLBR("Plant Breeding"),
    /**
     * Plant Breeding.
     */
    PLBRG("Plant Breeding"),
    /**
     * Horticulture Sciences.
     */
    PLHRT("Horticulture Sciences"),
    /**
     * Plant Pathology.
     */
    PLPPM("Plant Pathology"),
    /**
     * Plant Pathology.
     */
    PLPA("Plant Pathology"),
    /**
     * Plant Sciences.
     */
    PLSCI("Plant Sciences"),
    /**
     * Soil & Crop Sciences.
     */
    PLSCS("Soil & Crop Sciences"),
    /**
     * Performing and Media Arts.
     */
    PMA("Performing and Media Arts"),
    /**
     * Polish.
     */
    POLSH("Polish"),
    /**
     * Portuguese.
     */
    PORT("Portuguese"),
    /**
     * Psychology.
     */
    PSYCH("Psychology"),
    /**
     * Punjabi.
     */
    PUNJB("Punjabi"),
    /**
     * Religious Studies.
     */
    RELST("Religious Studies"),
    /**
     * Romanian.
     */
    ROMAN("Romanian"),
    /**
     * Romance Studies.
     */
    ROMS("Romance Studies"),
    /**
     * Russian.
     */
    RUSSA("Russian"),
    /**
     * Russian Literature.
     */
    RUSSL("Russian Literature"),
    /**
     * Sanskrit.
     */
    SANSK("Sanskrit"),
    /**
     * Sea Education Association
     */
    SEA("Sea Education Association"),
    /**
     * Society for Humanities.
     */
    SHUM("Society for Humanities"),
    /**
     * Sinhalese.
     */
    SINHA("Sinhalese"),
    /**
     * Science of Natural & Envir Sys.
     */
    SNES("Science of Natural & Envir Sys"),
    /**
     * Sanskrit Literature.
     */
    SNLIT("Sanskrit Literature"),
    /**
     * Sociology.
     */
    SOC("Sociology"),
    /**
     * Spanish.
     */
    SPAN("Spanish"),
    /**
     * Science & Technology Studies.
     */
    STS("Science & Technology Studies"),
    /**
     * Statistical Science.
     */
    STSCI("Statistical Science"),
    /**
     * Sumerian.
     */
    SUMER("Sumerian"),
    /**
     * Swahili.
     */
    SWAHL("Swahili"),
    /**
     * Systems Engineering.
     */
    SYSEN("Systems Engineering"),
    /**
     * Tagalog.
     */
    TAG("Tagalog"),
    /**
     * Tamil.
     */
    TAMIL("Tamil"),
    /**
     * Digital Technology & Practice.
     */
    TECH("Digital Technology & Practice"),
    /**
     * Thai.
     */
    THAI("Thai"),
    /**
     * Tibetan.
     */
    TIBET("Tibetan"),
    /**
     * Toxicology.
     */
    TOX("Toxicology"),
    /**
     * Ukrainian.
     */
    UKRAN("Ukrainian"),
    /**
     * Turkish.
     */
    TURK("Turkish"),
    /**
     * Urdu.
     */
    URDU("Urdu"),
    /**
     * Vet Med Clinical Sciences.
     */
    VETCS("Vet Med Clinical Sciences"),
    /**
     * Vet Med Microbiology.
     */
    VETMI("Vet Med Microbiology"),
    /**
     * Vet Medicine Molecular Medicine.
     */
    VETMM("Vet Medicine Molecular Medicine"),
    /**
     * Viticulture and Enology.
     */
    VIEN("Viticulture and Enology"),
    /**
     * Vietnamese.
     */
    VIET("Vietnamese"),
    /**
     * Visual Studies.
     */
    VISST("Visual Studies"),
    /**
     * Vet Med BioMedical Sciences.
     */
    VTBMS("Vet Med BioMedical Sciences"),
    /**
     * Vet Med Professional Curriculm.
     */
    VTMED("Vet Med Professional Curriculm"),
    /**
     * Population Med&Diagnostic Svc.
     */
    VTPMD("Population Med&Diagnostic Svc"),
    /**
     * Wolof.
     */
    WOLOF("Wolof"),
    /**
     * Writing Program.
     */
    WRIT("Writing Program"),
    /**
     * Yoruba.
     */
    YORUB("Yoruba"),
    /**
     * Zulu.
     */
    ZULU("Zulu");
    
    /**
     * A simple description of the code.
     */
    @NotNull
    private final String description;
    
    /**
     * Construct an {@code Subject}.
     *
     * @param description a simple description of the code.
     */
    Subject(@NotNull String description) {
        this.description = description;
    }
    
    /**
     * Obtain a simple description of the code.
     *
     * @return a simple description of the code.
     */
    @NotNull
    public String getDescription() {
        return description;
    }
}

File: kotlin/src/main/kotlin/api/cornell/data/classes/Subject.kt
package api.cornell.data.classes

/**
 * [Subject] is a collection of supported subjects value.
 *
 * @param description a simple description of the code.
 */
enum class Subject(val description: String) {
    /**
     * Asian American Studies.
     */
    AAS(description = "Asian American Studies"),
    /**
     * American Indian Studies.
     */
    AIS(description = "American Indian Studies"),
    /**
     * Applied Economics & Management.
     */
    AEM(description = "Applied Economics & Management"),
    /**
     * Applied & Engineering Physics.
     */
    AEP(description = "Applied & Engineering Physics"),
    /**
     * Agriculture Sciences.
     */
    AGSCI(description = "Agriculture Sciences"),
    /**
     * American Indian & Indigenous.
     */
    AIIS(description = "American Indian & Indigenous"),
    /**
     * Air Force Science.
     */
    AIRS(description = "Air Force Science"),
    /**
     * Akkadian.
     */
    AKKAD(description = "Akkadian"),
    /**
     * Agriculture & Life Sciences.
     */
    ALS(description = "Agriculture & Life Sciences"),
    /**
     * American Studies.
     */
    AMST(description = "American Studies"),
    /**
     * Animal Science.
     */
    ANSC(description = "Animal Science"),
    /**
     * Anthropology.
     */
    ANTHR(description = "Anthropology"),
    /**
     * Arabic.
     */
    ARAB(description = "Arabic"),
    /**
     * Architecture.
     */
    ARCH(description = "Architecture"),
    /**
     * Archaeology.
     */
    ARKEO(description = "Archaeology"),
    /**
     * Art.
     */
    ART(description = "Art"),
    /**
     * Art History.
     */
    ARTH(description = "Art History"),
    /**
     * Arts & Sciences.
     */
    AS(description = "Arts & Sciences"),
    /**
     * Asian Studies.
     */
    ASIAN(description = "Asian Studies"),
    /**
     * Africana Studies & Res Center.
     */
    ASRC(description = "Africana Studies & Res Center"),
    /**
     * Astronomy.
     */
    ASTRO(description = "Astronomy"),
    /**
     * Bosnian, Croatian, Serbian.
     */
    BCS(description = "Bosnian, Croatian, Serbian"),
    /**
     * Biological & Environmental Eng.
     */
    BEE(description = "Biological & Environmental Eng"),
    /**
     * Bengali.
     */
    BENGL(description = "Bengali"),
    /**
     * Animal Physiology & Anatomy.
     */
    BIOAP(description = "Animal Physiology & Anatomy"),
    /**
     * Ecology & Evolutionary Biology.
     */
    BIOEE(description = "Ecology & Evolutionary Biology"),
    /**
     * Biology: General Courses.
     */
    BIOG(description = "Biology: General Courses"),
    /**
     * Molecular Biology and Genetics.
     */
    BIOMG(description = "Molecular Biology and Genetics"),
    /**
     * Microbiology.
     */
    BIOMI(description = "Microbiology"),
    /**
     * Bio Medical Science.
     */
    BIOMS(description = "Bio Medical Science"),
    /**
     * Plant Biology.
     */
    BIOPL(description = "Plant Biology"),
    /**
     * Neurobiology & Behavior.
     */
    BIONB(description = "Neurobiology & Behavior"),
    /**
     * Biomedical Engineering.
     */
    BME(description = "Biomedical Engineering"),
    /**
     * Baker Program in Real Estate.
     */
    BPRE(description = "Baker Program in Real Estate"),
    /**
     * Biology & Society.
     */
    BSOC(description = "Biology & Society"),
    /**
     * Biometry & Statistics.
     */
    BTRY(description = "Biometry & Statistics"),
    /**
     * Burmese.
     */
    BURM(description = "Burmese"),
    /**
     * China & Asia Pacific Studies.
     */
    CAPS(description = "China & Asia Pacific Studies"),
    /**
     * Civil & Environmental Engr.
     */
    CEE(description = "Civil & Environmental Engr"),
    /**
     * Chemistry.
     */
    CHEM(description = "Chemistry"),
    /**
     * Chemical Engineering.
     */
    CHEME(description = "Chemical Engineering"),
    /**
     * Chinese.
     */
    CHIN(description = "Chinese"),
    /**
     * Chinese Literature.
     */
    CHLIT(description = "Chinese Literature"),
    /**
     * Classics.
     */
    CLASS(description = "Classics"),
    /**
     * Cognitive Science.
     */
    COGST(description = "Cognitive Science"),
    /**
     * College Scholar Program.
     */
    COLLS(description = "College Scholar Program"),
    /**
     * Comparative Literature.
     */
    COML(description = "Comparative Literature"),
    /**
     * Communication.
     */
    COMM(description = "Communication"),
    /**
     * City & Regional Planning.
     */
    CRP(description = "City & Regional Planning"),
    /**
     * Computer Science.
     */
    CS(description = "Computer Science"),
    /**
     * Crop & Soil Sciences.
     */
    CSS(description = "Crop & Soil Sciences"),
    /**
     * Czech.
     */
    CZECH(description = "Czech"),
    /**
     * Design & Environmental Analy.
     */
    DEA(description = "Design & Environmental Analy"),
    /**
     * Development Sociology.
     */
    DSOC(description = "Development Sociology"),
    /**
     * Dutch.
     */
    DUTCH(description = "Dutch"),
    /**
     * Earth & Atmospheric Sciences.
     */
    EAS(description = "Earth & Atmospheric Sciences"),
    /**
     * Electrical & Computer Engineering.
     */
    ECE(description = "Electrical & Computer Engineering"),
    /**
     * Economics.
     */
    ECON(description = "Economics"),
    /**
     * Education.
     */
    EDUC(description = "Education"),
    /**
     * English Language Support.
     */
    ELSO(description = "English Language Support"),
    /**
     * English.
     */
    ENGL(description = "English"),
    /**
     * Engineering Communications.
     */
    ENGRC(description = "Engineering Communications"),
    /**
     * Engineering Distribution.
     */
    ENGRD(description = "Engineering Distribution"),
    /**
     * Engineering General Interest.
     */
    ENGRG(description = "Engineering General Interest"),
    /**
     * Engineering Introduction.
     */
    ENGRI(description = "Engineering Introduction"),
    /**
     * English for International Stdt.
     */
    ENINT(description = "English for International Stdt"),
    /**
     * Engineering Management.
     */
    ENMGT(description = "Engineering Management"),
    /**
     * Entomology.
     */
    ENTOM(description = "Entomology"),
    /**
     * Envr and Sust Sciences.
     */
    ESS(description = "Envr and Sust Sciences"),
    /**
     * Food Science.
     */
    FDSC(description = "Food Science"),
    /**
     * Feminist,Gender,Sexuality Stdy.
     */
    FGSS(description = "Feminist,Gender,Sexuality Stdy"),
    /**
     * French.
     */
    FREN(description = "French"),
    /**
     * Fiber Science & Apparel Design.
     */
    FSAD(description = "Fiber Science & Apparel Design"),
    /**
     * German Studies.
     */
    GERST(description = "German Studies"),
    /**
     * Government.
     */
    GOVT(description = "Government"),
    /**
     * Graduate Research.
     */
    GRAD(description = "Graduate Research"),
    /**
     * Greek.
     */
    GREEK(description = "Greek"),
    /**
     * Hotel Administration.
     */
    HADM(description = "Hotel Administration"),
    /**
     * Human Development.
     */
    HD(description = "Human Development"),
    /**
     * Human Ecology Nondepartmental.
     */
    HE(description = "Human Ecology Nondepartmental"),
    /**
     * Hebrew.
     */
    HEBRW(description = "Hebrew"),
    /**
     * Hindi.
     */
    HINDI(description = "Hindi"),
    /**
     * History.
     */
    HIST(description = "History"),
    /**
     * Horticulture Sciences.
     */
    HORT(description = "Horticulture Sciences"),
    /**
     * Hungarian.
     */
    HUNGR(description = "Hungarian"),
    /**
     * Intl Agriculture & Rural Dev.
     */
    IARD(description = "Intl Agriculture & Rural Dev"),
    /**
     * ILR Human Resource Management.
     */
    ILRHR(description = "ILR Human Resource Management"),
    /**
     * ILR International & Comp Labor.
     */
    ILRIC(description = "ILR International & Comp Labor"),
    /**
     * ILR Interdepartmental.
     */
    ILRID(description = "ILR Interdepartmental"),
    /**
     * ILR Labor Economics.
     */
    ILRLE(description = "ILR Labor Economics"),
    /**
     * ILR Labor Relations, Law, Hist.
     */
    ILRLR(description = "ILR Labor Relations, Law, Hist"),
    /**
     * ILR Organizational Behavior.
     */
    ILROB(description = "ILR Organizational Behavior"),
    /**
     * ILR Social Statistics.
     */
    ILRST(description = "ILR Social Statistics"),
    /**
     * Independent Major.
     */
    IM(description = "Independent Major"),
    /**
     * Indonesian.
     */
    INDO(description = "Indonesian"),
    /**
     * Information Science.
     */
    INFO(description = "Information Science"),
    /**
     * Italian.
     */
    ITAL(description = "Italian"),
    /**
     * Japanese.
     */
    JAPAN(description = "Japanese"),
    /**
     * Japanese Literature.
     */
    JPLIT(description = "Japanese Literature"),
    /**
     * Jewish Studies.
     */
    JWST(description = "Jewish Studies"),
    /**
     * Kannada.
     */
    KANAD(description = "Kannada"),
    /**
     * Khmer.
     */
    KHMER(description = "Khmer"),
    /**
     * Korean.
     */
    KOREA(description = "Korean"),
    /**
     * Korean Literature.
     */
    KRLIT(description = "Korean Literature"),
    /**
     * Landscape Architecture.
     */
    LA(description = "Landscape Architecture"),
    /**
     * Latin American Studies.
     */
    LATA(description = "Latin American Studies"),
    /**
     * Latin.
     */
    LATIN(description = "Latin"),
    /**
     * Law.
     */
    LAW(description = "Law"),
    /**
     * Leadership.
     */
    LEAD(description = "Leadership"),
    /**
     * Lesbian, Gay, Bisexual, Trns Stdy.
     */
    LGBT(description = "Lesbian, Gay, Bisexual, Trns Stdy"),
    /**
     * Linguistics.
     */
    LING(description = "Linguistics"),
    /**
     * Latino Studies Program.
     */
    LSP(description = "Latino Studies Program"),
    /**
     * Mechanical & Aerospace Eng.
     */
    MAE(description = "Mechanical & Aerospace Eng"),
    /**
     * Mathematics.
     */
    MATH(description = "Mathematics"),
    /**
     * Medieval Studies.
     */
    MEDVL(description = "Medieval Studies"),
    /**
     * Military Science.
     */
    MILS(description = "Military Science"),
    /**
     * Modern Greek.
     */
    MOGRK(description = "Modern Greek"),
    /**
     * Materials Science & Engr.
     */
    MSE(description = "Materials Science & Engr"),
    /**
     * Music.
     */
    MUSIC(description = "Music"),
    /**
     * Grad Mgmt Acct.
     */
    NACCT(description = "Grad Mgmt Acct"),
    /**
     * Naval Science.
     */
    NAVS(description = "Naval Science"),
    /**
     * Grad Mgmt Business Admin.
     */
    NBA(description = "Grad Mgmt Business Admin"),
    /**
     * Grad Mgmt Business Admin CT
     */
    NBAT(description = "Grad Mgmt Business Admin CT"),
    /**
     * Executive Boardroom Electives.
     */
    NBAB(description = "Executive Boardroom Electives"),
    /**
     * Business Admin Electives EMBA.
     */
    NBAE(description = "Business Admin Electives EMBA"),
    /**
     * Grad Mgmt Business Admin NYT.
     */
    NBAY(description = "Grad Mgmt Business Admin NYT"),
    /**
     * Grad Mgmt Common Core.
     */
    NCC(description = "Grad Mgmt Common Core"),
    /**
     * Executive Boardroom Core.
     */
    NCCB(description = "Executive Boardroom Core"),
    /**
     * Grad Mgmt Common Core NYT.
     */
    NCCY(description = "Grad Mgmt Common Core NYT"),
    /**
     * Nepali.
     */
    NEPAL(description = "Nepali"),
    /**
     * Near Eastern Studies.
     */
    NES(description = "Near Eastern Studies"),
    /**
     * Grad Mgmt Individual Study.
     */
    NMI(description = "Grad Mgmt Individual Study"),
    /**
     * Graduate Management Research.
     */
    NRE(description = "Graduate Management Research"),
    /**
     * Nutritional Science.
     */
    NS(description = "Nutritional Science"),
    /**
     * Nuclear Science & Engineering.
     */
    NSE(description = "Nuclear Science & Engineering"),
    /**
     * Natural Resources.
     */
    NTRES(description = "Natural Resources"),
    /**
     * Op Research & Information Engr.
     */
    ORIE(description = "Op Research & Information Engr"),
    /**
     * Public Administration.
     */
    PADM(description = "Public Administration"),
    /**
     * Pali.
     */
    PALI(description = "Pali"),
    /**
     * Policy Analysis & Management.
     */
    PAM(description = "Policy Analysis & Management"),
    /**
     * Physical Education & Athletics.
     */
    PE(description = "Physical Education & Athletics"),
    /**
     * Persian.
     */
    PERSN(description = "Persian"),
    /**
     * Philosophy.
     */
    PHIL(description = "Philosophy"),
    /**
     * Physics.
     */
    PHYS(description = "Physics"),
    /**
     * Plant Biology.
     */
    PLBIO(description = "Plant Biology"),
    /**
     * Plant Breeding.
     */
    PLBR(description = "Plant Breeding"),
    /**
     * Plant Breeding.
     */
    PLBRG(description = "Plant Breeding"),
    /**
     * Horticulture Sciences.
     */
    PLHRT(description = "Horticulture Sciences"),
    /**
     * Plant Pathology.
     */
    PLPPM(description = "Plant Pathology"),
    /**
     * Plant Pathology.
     */
    PLPA(description = "Plant Pathology"),
    /**
     * Plant Sciences.
     */
    PLSCI(description = "Plant Sciences"),
    /**
     * Soil & Crop Sciences.
     */
    PLSCS(description = "Soil & Crop Sciences"),
    /**
     * Performing and Media Arts.
     */
    PMA(description = "Performing and Media Arts"),
    /**
     * Polish.
     */
    POLSH(description = "Polish"),
    /**
     * Portuguese.
     */
    PORT(description = "Portuguese"),
    /**
     * Psychology.
     */
    PSYCH(description = "Psychology"),
    /**
     * Punjabi.
     */
    PUNJB(description = "Punjabi"),
    /**
     * Religious Studies.
     */
    RELST(description = "Religious Studies"),
    /**
     * Romanian.
     */
    ROMAN(description = "Romanian"),
    /**
     * Romance Studies.
     */
    ROMS(description = "Romance Studies"),
    /**
     * Russian.
     */
    RUSSA(description = "Russian"),
    /**
     * Russian Literature.
     */
    RUSSL(description = "Russian Literature"),
    /**
     * Sanskrit.
     */
    SANSK(description = "Sanskrit"),
    /**
     * Sea Education Association
     */
    SEA(description = "Sea Education Association"),
    /**
     * Society for Humanities.
     */
    SHUM(description = "Society for Humanities"),
    /**
     * Sinhalese.
     */
    SINHA(description = "Sinhalese"),
    /**
     * Science of Natural & Envir Sys.
     */
    SNES(description = "Science of Natural & Envir Sys"),
    /**
     * Sanskrit Literature.
     */
    SNLIT(description = "Sanskrit Literature"),
    /**
     * Sociology.
     */
    SOC(description = "Sociology"),
    /**
     * Spanish.
     */
    SPAN(description = "Spanish"),
    /**
     * Science & Technology Studies.
     */
    STS(description = "Science & Technology Studies"),
    /**
     * Statistical Science.
     */
    STSCI(description = "Statistical Science"),
    /**
     * Sumerian.
     */
    SUMER(description = "Sumerian"),
    /**
     * Swahili.
     */
    SWAHL(description = "Swahili"),
    /**
     * Systems Engineering.
     */
    SYSEN(description = "Systems Engineering"),
    /**
     * Tagalog.
     */
    TAG(description = "Tagalog"),
    /**
     * Tamil.
     */
    TAMIL(description = "Tamil"),
    /**
     * Digital Technology & Practice.
     */
    TECH(description = "Digital Technology & Practice"),
    /**
     * Thai.
     */
    THAI(description = "Thai"),
    /**
     * Tibetan.
     */
    TIBET(description = "Tibetan"),
    /**
     * Toxicology.
     */
    TOX(description = "Toxicology"),
    /**
     * Ukrainian.
     */
    UKRAN(description = "Ukrainian"),
    /**
     * Turkish.
     */
    TURK(description = "Turkish"),
    /**
     * Urdu.
     */
    URDU(description = "Urdu"),
    /**
     * Vet Med Clinical Sciences.
     */
    VETCS(description = "Vet Med Clinical Sciences"),
    /**
     * Vet Med Microbiology.
     */
    VETMI(description = "Vet Med Microbiology"),
    /**
     * Vet Medicine Molecular Medicine.
     */
    VETMM(description = "Vet Medicine Molecular Medicine"),
    /**
     * Viticulture and Enology.
     */
    VIEN(description = "Viticulture and Enology"),
    /**
     * Vietnamese.
     */
    VIET(description = "Vietnamese"),
    /**
     * Visual Studies.
     */
    VISST(description = "Visual Studies"),
    /**
     * Vet Med BioMedical Sciences.
     */
    VTBMS(description = "Vet Med BioMedical Sciences"),
    /**
     * Vet Med Professional Curriculm.
     */
    VTMED(description = "Vet Med Professional Curriculm"),
    /**
     * Population Med&Diagnostic Svc.
     */
    VTPMD(description = "Population Med&Diagnostic Svc"),
    /**
     * Wolof.
     */
    WOLOF(description = "Wolof"),
    /**
     * Writing Program.
     */
    WRIT(description = "Writing Program"),
    /**
     * Yoruba.
     */
    YORUB(description = "Yoruba"),
    /**
     * Zulu.
     */
    ZULU(description = "Zulu")
}
